tock/tock/settings/dev.py

```diff
@@ -10,9 +11,9 @@
         'ENGINE': 'django.db.backends.postgresql_psycopg2', # Add 'postgresql_psycopg2', 'mysql', 'sqlite3' or 'oracle'.
         'NAME': 'tock',                      # Or path to database file if using sqlite3.
         # The following settings are not used with sqlite3:
-        'USER': 'tock',
-        'PASSWORD': 'tock',
-        'HOST': 'localhost',                      # Empty for localhost through domain sockets or           '127.0.0.1' for localhost through TCP.
+        'USER': os.environ.get('DATABASE_USER'),
```

Comment:
No. These are for the postgres image to set the default user and database. 
And for psql to do the same.

The postgres image will create a new user `tock` with the password `tock` and a new database `tock` for this user.